bin/load_test

```diff
@@ -35,14 +35,20 @@ if [ $1 ]
 then
   test_to_run="$1"
   target="${TARGET_HOST:-http://localhost:3000}"
+  num_clients=${NUM_CLIENTS:-3}
+  num_requests=${NUM_REQUESTS:-100}
+  hatch_rate=${HATCH_RATE:-1}
 
   brew_install_or_upgrade 'libevent'
   source /usr/local/bin/virtualenvwrapper.sh
   mkvirtualenv locust && \
     workon locust &&  \
     pip install -r scripts/load_testing/requirements.txt && \
-    locust -f scripts/load_testing/$test_to_run.py -H "$target" --no-web -c 3 -n 27 -r 1
+    echo "running locust with concurrency of $num_clients, new user hatch rate"\
+    "of $hatch_rate, and a total number of $num_requests requests" && \
+    locust -f scripts/load_testing/$test_to_run.py -H "$target" --no-web \
+    -c $num_clients -n $hatch_rate -r $num_requests
   deactivate
 else
-  echo "Usage: load_test [name_of_file]"
+  echo "Usage: NUM_CLIENTS=3 NUM_REQUESTS=100 HATCH_RATE=1 load_test [name_of_file]"
```

Comment:
The old format would still work, right? So could we make it more explicit that the args are optional?

Reply:
Yeah, I wasn't sure how to make it explicit that those are optional args. WDYT?

Reply:
I think I'd just have two Usage lines:
```
echo "Usage:"
echo "load_test [name_of_file]"
echo "or, if you'd like to set the arguments"
echo "NUM_CLIENTS=3 NUM_REQUESTS=100 HATCH_RATE=1 load_test [name_of_file]"

Reply:
That works, I'll add that and fix a minor bug (-r and -n were reversed)